perl-package/AI-MXNetCAPI/mxnet.i

```diff
@@ -0,0 +1,1558 @@
+%module "AI::MXNetCAPI"
```

Comment:
I'm not quite sure what you mean.
For example
```
+/*!
 + * \brief create a NDArray with specified shape
 + * \param shape the pointer to the shape
 + * \param ndim the dimension of the shape
 + * \param dev_type device type, specify device we want to take
 + * \param dev_id the device id of the specific device
 + * \param delay_alloc whether to delay allocation until
 + *    the narray is first mutated
 + * \param out the returning handle
 + * \return 0 when success, -1 when failure happens
 + */
 +int MXNDArrayCreate(const mx_uint *in,
 +                              mx_uint ndim,
 +                              int dev_type,
 +                              int dev_id,
 +                              int delay_alloc,
 +                              NDArrayHandle *out);
```
These are copy-pasted code. Can you include them instead?